internal/engine/upper.go

```diff
@@ -92,13 +92,15 @@ func (u Upper) RunHud(ctx context.Context) error {
 	return err
 }
 
-func (u Upper) CreateManifests(ctx context.Context, manifests []model.Manifest, watchMounts bool) error {
-	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-Up")
+func (u Upper) CreateManifests(ctx context.Context, manifests []model.Manifest,
+	globalYAML model.YAMLManifest, watchMounts bool) error {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "Up")
 	defer span.Finish()
 
 	u.store.Dispatch(InitAction{
-		WatchMounts: watchMounts,
-		Manifests:   manifests,
+		WatchMounts:        watchMounts,
+		Manifests:          manifests,
+		GlobalYAMLManifest: globalYAML,
```

Comment:
Wait- I'm confused: in this code already in master? https://github.com/windmilleng/tilt/blob/6da59ebe249922c9758dc3d40cd59bba23eaaac4/internal/engine/upper.go#L103